### DISCUSSION TITLE
Stripe checkout with Nuxt3

### DISCUSSION THREAD

**Discussion by SamG92** [nuxt/nuxt#19059] [Questions] [answered]
Hello,

I'm quite new to Nuxt3 and would like to integrate a recurring payment solution to my app.
I opened a Stripe account and have been looking for an example of Nuxt3 integration of low code "stripe checkout" solution but could not find any.

Do you know where I could find any working code example I could start with?

Thanks a lot!
Samuel

**Comment by Baroshem** [comment, marked as answer]
Hey, 

Check out the amazing video tutorial by my friend @rohrig :)

https://www.youtube.com/watch?v=xkGud5gMiNI

**Reply by SamG92** [reply]
Thank you so much, you saved my day!!!
